instruqt/delivering-as-an-appliance/02-customize-for-your-app/assignment.md

```diff
@@ -16,6 +20,184 @@ timelimit: 300
 ---
 
 The Kubernetes appliance installation is one of the first things
-your customer will experience about your application. Replicated
-provides a few options to customize the appliance installer. Let's
-use these to tailor it for the Slackernews application.
+your customer will experience about your application. The way we've implemented
+it so far uses a lot of defaults. We didn't try to install it yet, but if we
+had, we'd land on a page like this:
+
+![Appliance login page without branding](../assets/unbranded-admin-console.png)
+
+This definitely violates our brand standards!
+
+Don't worry, Replicated provides a few options to customize the appliance
+installer. The first customizations we're going to look at will make it clear
+which application we're installing.
+
+Adding a Name and Logo to the Appliance
+=======================================
+
+We've already looked at two of the manifests we use for configuring the
+appliance experience: one for the embedded Kubernetes cluster and one for the
+Slackernews Helm chart. There are other manifests as well. We're going to look
+at the `Application` manifest, which describes the application being installed.
+
+```
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: slackernews
+spec:
+  title: SlackerNews
+  icon: "https://uploads-ssl.webflow.com/6310ad0e6a18aa1620da6ae8/6330e04f42bc6a7ba03b4725_snicon.png"
+```
+
+There are other things we can configure in this file as well, but for this lab
+we'll focus just on the title and the icon. For other possible properties, take
+a look at the [reference
+documentation](https://docs.replicated.com/reference/custom-resource-application).
+Not all of the possible values apply for the Kubernetes appliance experience.
+
+The appliance installation process also supports another resource that
+describes application metadata. The resource is based defined by the
+Kubernetes Apps Special Interest Group (SIG) and is also called `Application`.
+This `Application` resource is not unique to the Replicated Platform. It's
+also not required for the appliance installation experience.
+
+We'll start with adding the basic Replicated `Application` resource for basic
+branding, then we'll expand on it a bit. We'll create a SIG Application
+resource as part of that expansion.
+
+### Add the Application Manifest
+
+Let's go back to the "Release Editor" tab and add a new file for the
+application metadata. Create a new file in the `release` directory named
+`replicated-app.yaml` and copy/paste the contents from the example above into
```

Comment:
When I saw the copy for this file above, I thought I needed to create the file. Then I didn't have to until here, and I had to scroll back up to copy it. I think it's clearer to have the content right here when I need to copy them, and not earlier. IMO the link to the docs shows what the file looks like as you're reading through that earlier part.